.github/pull_request_template.md

```diff
@@ -18,7 +18,7 @@ Fixes # (issue)
 
 ## Checklist
 
-- [ ] I have a descriptive title for my PR
-- [ ] I have linked this PR to the corresponding issue. [See how to do that here.](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
-- [ ] I have run the project locally and reviewed the code changes
-- [ ] My changes generate no new warnings
```

Comment:
Why is this file changed. It has nothing to do with #107 issue.
Please clarify or undo changes for this file.

Reply:
I was having trouble trying to undo the ticked boxes on the last pr and made the change. If there is a way to delete this then that would be great

Reply:
you can simply edit `pull_request_template.md` file

just make it like it was before your last commit.

and then push such changes to this PR.

Reply:
I already pushed this change yesterday and it should be in the commit section 'update pull_request_template.md

Reply:
I get it. but the thing is
If you will take look at the `files changed` tab right now - you will see that you did not undo changes exactly right. 4 lines - 21, 22, 23, 24 - are still considered changed. Originally there was a space between the brackets. In your last version of code there is no space between the brackets.
So add such spaces and push it to this PR.